applications/feedback_sentiment_analyzer/cdk/README.md

```diff
@@ -0,0 +1,59 @@
+# AWS CDK for FSA
+
+Deploy the Feedback Sentiment analyzer with the AWS Cloud Development Kit (CDK).
+
+## Prerequisites
+
+- Install NodeJS 18+
+- Install AWS CDK
+  - https://docs.aws.amazon.com/cdk/v2/guide/cli.html
+- AWS access key & secret access key for IAM user with permissions to create resources listed above
+  - https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config
+
+## Set variables
+
+Export these variables:
+
+| Name      | Usage                                                    |
+| --------- | -------------------------------------------------------- |
+| APP_NAME  | Short one-word name to identify this stack.              |
```

Comment:
On its own, APP_* is good, but I anticipate our team will be moving between these stacks. I worry that we'll accidentally use APP_NAME from PAM and then reuse it in FSA... so I think we should keep the env variables as {TLA}_NAME where TLA is PAM, FSA, etc. (They can all be APP_* in the code, but for env they should be unique per app).